cookbooks/app/metadata.rb

```diff
@@ -17,3 +17,4 @@
 depends 'lb'
 depends 'env_vars'
 depends 'cdn_distribution'
+depends 'puma'
```

Comment:
Do we need this line? Just wondering why the other app server recipes (unicorn, passenger) works without this.

Reply:
Not sure why this it is required.

Reply:
There is no requirement for this addition IMHO, I'm deleting this file